src/typecheck/mk_typewrapper.rs

```diff
@@ -1,14 +1,14 @@
 //! Helpers for building `TypeWrapper`s.
```

Comment:
Maybe we should rename `mk_typewrapper.rs` to something like `mk_uniftype.rs`? We got rid of the `TypeWrapper` name in favor of `UnifType`.

Reply:
Yes, this is what I meant in the first bullet point of the Todo (future work) section, but it's not very clear. As it's not needed, and the diff is quite large, I propose to postpone some cosmetic changes like this to future PRs.